src/Main.jl

```diff
@@ -59,7 +60,7 @@ module MainLoop
 			this.selectedEntityIndex = -1
 			this.selectedTextBoxIndex = -1
 			this.selectedEntityUpdated = false
-			this.screenDimensions = C_NULL
+			this.screenDimensions = C_NULL # Why is this a C_NULL by default? why not Math.Vector2(0,0)?

```

Comment:
You'll see a lot of head-scratchers here. This is my lack of knowledge of Julia along with trying to move quickly. I didn't look, but there may be some logic that checks for a C_NULL before doing anything. If not, we can probably change this to Math.Vector2(0,0)

Reply:
I'll check before changing anything!